openfl-workspace/torch_cnn_mnist/code/pt_cnn.py

```diff
@@ -42,10 +43,23 @@ def __init__(self, device='cpu', **kwargs):
         super().__init__(device=device, **kwargs)
 
         self.num_classes = self.data_loader.num_classes
-        self.init_network(device=self.device, **kwargs)
-        self._init_optimizer()
-        self.loss_fn = cross_entropy
-        self.initialize_tensorkeys_for_functions()
+
+        path = 'model/saved_model.pth'
+        if os.path.exists(path):
+            # Load the previously trained model
+            self.load_native(filepath=path)
+            if kwargs['loss']:
+                self.loss_fn = kwargs['loss']
```

Comment:
This will set `self.loss_fn` to a string. You should add additional code to check if the provided function is found within `torch.nn`, and then resolve it with `getattr`